rust/CMakeLists.txt

```diff
@@ -1,2 +1,7 @@
 add_subdirectory (workspace)
-add_subdirectory (chcache)
+
+option(ENABLE_CHCACHE "Enable chcache" ${ENABLE_RUST})
```

Comment:
I just realized we have a similar check in `rust/chcache/CMakeLists.txt`, so currently this doesn't make much sense. I will fix it.